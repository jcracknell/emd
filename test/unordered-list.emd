* tight
* list
* items

Loose list:

* loose
* list

* items

* multiple
lines

* item

    continues

// asterix with no trailing space is emphasis at paragraph start
*emphasized text
