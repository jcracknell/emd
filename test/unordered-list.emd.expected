#! emd.cs.Conversion.StructuralRenderer, emd.cs
(Document
	(UnorderedList
		(UnorderedListItem
			(Text "tight")
		)
		(UnorderedListItem
			(Text "list")
		)
		(UnorderedListItem
			(Text "items")
		)
	)
	(Paragraph
		(Text "Loose")
		(Space)
		(Text "list:")
	)
	(UnorderedList
		(UnorderedListItem
			(Paragraph
				(Text "loose")
			)
		)
		(UnorderedListItem
			(Paragraph
				(Text "list")
			)
		)
		(UnorderedListItem
			(Paragraph
				(Text "items")
			)
		)
		(UnorderedListItem
			(Paragraph
				(Text "multiple")
				(Space)
				(Text "lines")
			)
		)
		(UnorderedListItem
			(Paragraph
				(Text "item")
			)
			(Paragraph
				(Text "continues")
			)
		)
	)
	(Paragraph
		(Emphasis
			(Text "emphasized")
			(Space)
			(Text "text")
		)
	)
)
